Doc/library/pathlib.rst

```diff
@@ -1147,7 +1147,10 @@ call fails (for example because the path doesn't exist).
 
    Rename this file or directory to the given *target*, and return a new Path
    instance pointing to *target*.  On Unix, if *target* exists and is a file,
-   it will be replaced silently if the user has permission.
+   it will be replaced silently if the user has permission. The operation may
+   fail on some Unix flavors if *src* and *dst* are on different filesystems.
+   If successful, the renaming will be an atomic operation (this is a POSIX requirement).
```

Comment:
Do these two lines only apply to Unix? If so, I would phrase similar to the Windows specific line below.

A

Reply:
I do not know, what happens on Windows. I have checked, that `Path.rename` is implemented with `os.rename` and checked its documentation. So, I believe, that this line is POSIX specific. I have copied it from `os.rename' docs. Should it be written:

> On Unix, if successful, the renaming will be an atomic operation (this is a POSIX requirement).

Should I change it in `os.rename' docs also?